src/pages/about/zetachain.mdx

```diff
@@ -1,42 +1,43 @@
 # What is ZetaChain?
 
-ZetaChain is the foundational, public blockchain that enables omnichain, generic
-smart contracts and messaging between _any_ blockchain. It solves the problems
-of “cross-chain” and “multi-chain” and aims to open the crypto and global
-financial ecosystem to anyone. ZetaChain envisions and supports a truly fluid,
-multi-chain crypto ecosystem, where users and developers can move between and
-appreciate the benefits of any blockchain: payments, DeFi, liquidity, games,
-art, social graphs, performance, security, privacy, and so on.
+ZetaChain is the foundational, public blockchain that enables omnichain smart
+contracts and messaging between any blockchain. It solves the problems of
+“cross-chain” and “multi-chain” and aims to open the crypto and global financial
+ecosystem to anyone. ZetaChain envisions and supports a truly fluid, multi-chain
+crypto ecosystem, where users and developers can move between and appreciate the
+benefits of any blockchain: payments, DeFi, liquidity, games, art, social
+graphs, performance, security, privacy, and more. As the first Universal EVM,
+ZetaChain abstracts chains, assets, and data, orchestrating them to offer a
+seamless and comprehensive user experience across all connected networks.
 
 View the whitepaper [here](https://zetachain.com/whitepaper.pdf) for a closer
 look at ZetaChain's background and architecture.
 
-## Key features
+## Key Features
 
-### Decentralized and public
+### Decentralized and Public Blockchain
 
 ZetaChain is a decentralized and public blockchain network. It is built on
-Cosmos SDK and Tendermint Consensus. While many cross-chain solutions like
-bridges have to vary, often centralized trust models that have a track record
-for being susceptible to exploits and hacks, ZetaChain is a Proof-of-Stake
-blockchain, where all transactions and activity on the platform -- even
-cross-chain transactions -- are fully transparent, verifiable, and function in a
-trust-minimized manner.
+Cosmos SDK and Comet BFT consensus. While many cross-chain solutions like
+bridges often have centralized trust models with track record of exploits and
+hacks, ZetaChain is a Proof-of-Stake blockchain, where all transactions and
+activity on the platform -- even cross-chain transactions -- are fully
+transparent, verifiable, and function in a trust-minimized manner.
 
 ### Hyper-connected nodes
```

Comment:
Could we potentially modify this section to change this title to Chain Abstraction Framework?